TranslatorReasonerAPI.yaml

```diff
@@ -131,11 +131,18 @@ components:
       required:
         - message
     Message:
+      description: >-
+        The message object holds the main content of a Query or a Response.
+        This includes the query configuration in the query_graph property,
+        any answers in the results property, and a mapping of response nodes
+        and edges to their QueryGraph counterparts in the knowledge_graph
+        property. The content of these properties is context-dependent
```

Comment:
> any answers in the results property, and a mapping of response nodes
>         and edges to their QueryGraph counterparts in the knowledge_graph
>         property

This wording is a little confusing to me. It sounds like the results property and the mapping of response nodes and edges are separate things, when I believe you're referring to the same thing.